core/flyout.js

```diff
@@ -406,6 +414,38 @@ Blockly.Flyout.prototype.show = function(xmlList) {
     }
   }
 
+  this.layoutBlocks_(blocks, gaps, margin);
+
+  // IE 11 is an incompetant browser that fails to fire mouseout events.
+  // When the mouse is over the background, deselect all blocks.
+  var deselectAll = function(e) {
+    var blocks = this.workspace_.getTopBlocks(false);
+    for (var i = 0, block; block = blocks[i]; i++) {
+      block.removeSelect();
+    }
+  };
+  this.listeners_.push(Blockly.bindEvent_(this.svgBackground_, 'mouseover',
+      this, deselectAll));
+
+  this.width_ = 0;
+  this.reflow();
+
+  this.filterForCapacity_();
+
+  // Fire a resize event to update the flyout's scrollbar.
+  Blockly.fireUiEventNow(window, 'resize');
+  this.reflowWrapper_ = this.reflow.bind(this);
+  this.workspace_.addChangeListener(this.reflowWrapper_);
+};
+
+/**
+ * Lay out the blocks in the flyout.
+ * @param {!Array<Blockly.BlockSvg>} blocks The blocks to lay out.
```

Comment:
FYI: Common error, should be {!Array.<Blockly.BlockSvg>}
https://google.github.io/styleguide/javascriptguide.xml?showone=JavaScript_Types#JavaScript_Types
I'll fix this after merge.